Client/FeatureFlags/FeatureFlagsManager.swift

```diff
@@ -146,7 +146,7 @@ class FeatureFlagsManager {
 
         let historyHighlights = FlaggableFeature(withID: .historyHighlights,
                                                  and: profile,
-                                                 enabledFor: [.developer])
+                                                 enabledFor: [.release, .beta, .developer])
```

Comment:
we are good to put it in release? 

do we need this behind nimbus flag and if so do we need to do anything in the new nimbus layer?

Reply:
cc @nishant2718 @yoanarios

Reply:
@nbhasin2 yes Daniela ask us to enable for release. As for the nimbus flag I'm not sure if we need something else in the `FxHomeHistoryHightlightsViewModel` `isEnabled` I see only the check for `isFeatureActiveForBuild` and `userPreferenceFor`. What do you think?

Reply:
I’d recommend rebasing from main and making sure nimbus layer has all the required changes for your .isEnabled flag. Let me find the example @yoanarios

Reply:
cc @electricRGB